### ISSUE TITLE
Add/test nvidia driver workaround config to regolith picom config

### ISSUE THREAD

**Issue by kgilmer** [regolith-linux/regolith-desktop#448]
Based on https://github.com/regolith-linux/regolith-desktop/issues/116

@ankitson has found some flags that prevent some nvidia driver issues.  The task is to add those and do some local testing to verify no regressions. If good, maintain flags in config until next fix release of picom.

**Comment by potaito** [comment]
As I mentioned in the other issue as well, this solves the problem for me as well for the following configuration:

Lenovo X1 Extreme (2nd gen)
GPU: Nvidia Geforce GTX1650
Regolith Version 1.4.1.

It makes sense, that this flag helps. According to description, it may force additional draw calls:
```
--xrender-sync-fence
  Additionally use X Sync fence to sync clients' draw calls. Needed
  on nvidia-drivers with GLX backend for some users.
```

One of the problems I had was, that after entering an application like `nano` or `vim`, it would still show the previous terminal text. Same after exiting the application, where it would still show Nano's toolbar for example. Hitting `CTRL+L` a bunch of times would force a redraw, which fixed the problem for a second.

**Comment by moritzheiber** [comment]
This new flag has shipped with the latest release 1.5.1 and is enabled by default.
